torch/quantization/quantize_fx.py

```diff
@@ -253,6 +253,18 @@ def convert_fx(graph_module, inplace=False, debug=False, convert_custom_config_d
         `debug`: flag for producing a debug friendly model (preserve weight attribute)
         `convert_custom_config_dict`: dictionary for custom configurations for convert function:
         convert_custom_config_dict = {
+          # addtional object (module/operator) mappings that will overwrite the default
+          # module mappingn
+          "additional_object_mapping": {
+             "static": {
+                FloatModule: QuantizedModule,
+                float_op: quantized_op
+             },
+             "dynamic": {
+                FloatModule: DynamicallyQuantizedModule,
+                float_op: dynamically_quantized_op
+             },
```

Comment:
how would custom mappings for QAT, and/or for models which have some parts quantized dynamically and statically fit?

Reply:
you mean additional qat module mappings? it's added in the next PR. it's kind of independent from these, it happens before we insert observer, e.g. we swap nn.Conv2d to nn.qat.Conv2d

"static" and "dynamic" here are used during convert, e.g. nn.qat.Conv2d to nn.quantized.Conv2d